Build script error

`Error: Cannot find module './lib/file'`

```
yarn install v1.2.1
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ ./scripts/build.js
module.js:538
    throw err;
    ^

Error: Cannot find module './lib/file'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/jnordberg/Development/steemit/steem-crypto-js/scripts/build.js:8:12)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
error Command failed with exit code 1.
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,2 @@
-lib
+/lib
 node_modules

--- a/scripts/lib/file.js
+++ b/scripts/lib/file.js
@@ -1,0 +1,129 @@
+/* eslint-env node */
+
+var fs = require('fs');
+var path = require('path');
+
+var readdir = promisify(fs.readdir);
+var stat = promisify(fs.stat);
+var copyFile = promisify(fs.copyFile);
+var writeFile = promisify(fs.writeFile);
+var mkdir = promisify(fs.mkdir);
+
+exports.promisify = promisify;
+exports.recursiveReaddir = recursiveReaddir;
+exports.read = promisify(fs.readFile);
+exports.copy = copy;
+exports.output = output;
+exports.rebasePath = rebasePath;
+exports.mkdirP = mkdirP;
+exports.concat = concat;
+
+function concat(files) {
+  return Promise.all(
+    files.map(function(filename) {
+      return exports.read(filename);
+    })
+  ).then(function(filesContents) {
+    return Buffer.concat(filesContents).toString();
+  });
+}
+
+function promisify(originalFn) {
+  return function() {
+    var scope = this;
+    var args = Array.prototype.slice.call(arguments, 0);
+    return new Promise(function(resolve, reject) {
+      try {
+        originalFn.apply(
+          scope,
+          [].concat(args, function(err) {
+            if (err) {
+              reject(arguments[0]);
+            } else if (arguments.length == 2) {
+              resolve(arguments[1]);
+            } else {
+              resolve(args.slice(1));
+            }
+          })
+        );
+      } catch (e) {
+        reject(e);
+      }
+    });
+  };
+}
+
+function copy(from, to) {
+  return mkdirP(path.dirname(to)).then(function() {
+    return copyFile(from, to);
+  });
+}
+
+function output(filePath, content) {
+  return mkdirP(path.dirname(filePath)).then(function() {
+    return writeFile(filePath, content);
+  });
+}
+
+function mkdirP(dirpath) {
+  var np = path.normalize(path.relative(process.cwd(), dirpath));
+  var bits = np.split(path.sep);
+
+  return bits.reduce(function(promise, bit, i) {
+    return promise.then(function() {
+      var nextPath = path.join.apply(path, bits.slice(0, i + 1));
+      return mkdir(nextPath).catch(function(e) {
+        if (e.code !== 'EEXIST') {
+          throw e;
+        }
+      });
+    });
+  }, Promise.resolve());
+}
+
+function recursiveReaddir(filePath, extension) {
+  return readdir(filePath)
+    .then(function(files) {
+      return Promise.all(
+        files.map(function(file) {
+          return stat(filePath + '/' + file);
+        })
+      ).then(function(stats) {
+        return Promise.all(
+          stats.map(function(stat, i) {
+            if (stat.isDirectory()) {
+              return recursiveReaddir(filePath + '/' + files[i]);
+            } else {
+              return Promise.resolve(filePath + '/' + files[i]);
+            }
+          })
+        );
+      });
+    })
+    .then(function(results) {
+      var allFiles = [].concat.apply([], results).map(path.normalize);
+      if (extension) {
+        return allFiles.filter(function(s) {
+          var extname = path.extname(s);
+          return (
+            extname.indexOf(extension) === extname.length - extension.length
+          );
+        });
+      }
+      return allFiles;
+    });
+}
+
+function rebasePath(src, dest, extname) {
+  extname = extname || path.extname(src);
+  var splicedSrcDirname = path.dirname(
+    path
+      .normalize(src)
+      .split(path.sep)
+      .slice(1)
+      .join(path.sep)
+  );
+  var srcBasename = path.basename(src).replace(/\.[^.]+$/, '');
+
+  return path.join(dest, splicedSrcDirname, srcBasename) + extname;
+}

--- a/scripts/lib/util.js
+++ b/scripts/lib/util.js
@@ -1,0 +1,36 @@
+/* eslint-env node */
+
+exports.camelCase = camelCase;
+exports.filterObject = filterObject;
+
+function camelCase(str) {
+  return str
+    .replace(/::\w/g, function(match) {
+      return '.' + match[2];
+    })
+    .replace(/_\w/g, function(match) {
+      return match[1].toUpperCase();
+    })
+    .replace(/\.([\w\d]+)$/, function(match) {
+      return '.' + match[1].toUpperCase() + match.slice(2);
+    });
+}
+
+function filterObject(obj, name) {
+  if (Array.isArray(obj)) {
+    return obj.map(function(el) {
+      return filterObject(el, name);
+    });
+  } else if (typeof obj === 'object' && obj !== null) {
+    var result = {};
+    for (var key in obj) {
+      if (name === key) {
+        continue;
+      }
+      result[key] = filterObject(obj[key], name);
+    }
+    return result;
+  } else {
+    return obj;
+  }
+}

